client/common/cmdline.c

```diff
@@ -2513,20 +2513,22 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 	}
 	while ((arg = CommandLineFindNextArgumentA(arg)) != NULL);
 
-	free(settings->Username);
-
-	if (!settings->Domain && user)
+	if (!settings->Username && user)
```

Comment:
This check is no good idea. If I want to override the username provided via RDP file this check will prevent it. A simple ```if (user)``` should do the trick.